app/src/main/java/com/xmartlabs/daydreaming/ui/VideoFragment.java

```diff
@@ -131,12 +143,14 @@ void onClickedMuteButton() {
 
   @OnClick(R.id.next_button)
   void onClickedNextButton() {
-    //TODO go to next video
+    currentIndex++;
+    playVideo(currentIndex > videos.size() - 1 ? 0 : currentIndex);
```

Comment:
Move the `currentIndex > videos.size() - 1 ? 0 : currentIndex` to it's own method.
Maybe this check should be done when setting a new value